view/frontend/templates/script/sentry.phtml

```diff
@@ -26,79 +28,86 @@ $remoteFile = sprintf(
 ?>
 
 <script src="<?= /** @noEscape */$remoteFile ?>" crossorigin="anonymous"></script>
-<script>
+<?php
+// The following script can be omitted in which case the
+// `Chessio_Matomo/js/tracker' component will inject the tracker script instead.
+// However that might cause the tracker script to miss the `DOMContentLoaded'
+// event which breaks the link tracking feature.
+$scriptString = <<<script
```

Comment:
While necessary this reduces readability quite a lot, Is there some other way we can do this?
Like loading a separate phtml file with the script as a string (, Possibly stripping the script tag) and passing that to the `secureRender->renderTag` function, so we can still keep regular script structure.